README.md

```diff
@@ -18,7 +18,7 @@ Welcome to the docs! This repo is used to build [the decisive engine docs site](
 ```sh
 cargo install mdbook mdbook-toc
 ```
-
+- Additionally, we are using the *catppuccin* theme. If you would like to install the assets from source and modify. Follow [install instructions](https://github.com/catppuccin/mdBook?tab=readme-ov-file).
```

Comment:
Any reason to do this other than look?

Reply:
look and it has some elements that will display better